src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ CHANGELOG
  * removed `Symfony\Component\HttpKernel\HttpCache\EsiResponseCacheStrategyInterface`
  * removed `Symfony\Component\HttpKernel\Log\LoggerInterface`
  * removed `Symfony\Component\HttpKernel\Log\NullLogger`
+ * made the `RequestStack` parameter required in the constructor of the following classes, which also changed the parameter order:
```

Comment:
The order of the parameters was already changed for Symfony 2.8 and should therefore not be described here.